manifests/experiences/twitter/twitter.json

```diff
@@ -41,7 +41,8 @@
     },
     {
       "customPipeline": "targetingTree",
-      "visualizations": ["sunburst-targeting"],
+      "visualizations": ["Sunburst.vue"],
+      "graphId": "tcTop",
```

Comment:
It's not completely coherent to have the graphId like this because the block may contain multiple visualizations. Possible solutions:
- Generate a random ID inside Sunburst.vue
- Decide that 1 block should not contain more than 1 viz
- Leave it like this until it's a real problem

Reply:
I'm in favor of generating the Id inside of Sunburst.vue, it doesn't need to be random, it could use a counter or something.

Element ids are supposed to be unique on the page, and it's possible that we end up having two instances of the same experience next to each other on the same page.

Reply:
Thanks for the review, I just find out that each component has a unique ID which can be accessed as this._uid, I will use this ID instead